README.md

```diff
@@ -18,10 +18,71 @@ To make your life easier, we already have a running ohsome API instance on our s
 
 If you want to contribute to the code base of the ohsome API, please follow the guideline and hints in the upcoming sections.
 
+## Examples
+
+The following blog posts describe analyses which were using the ohsome API:
+   
+* [Farm shops are ohsome](https://heigit.org/farm-shops-are-ohsome/)
+* [Visualizing the historical OSM evolution of your city](https://heigit.org/how-to-become-ohsome-part-1-visualizing-the-historical-evolution-of-osm-buildings-of-your-city/) 
+* [Exploring OSM history: the example of health related amenities](https://heigit.org/exploring-osm-history-the-example-of-health-related-amenities/)
+* several posts of the [how to become ohsome](https://heigit.org/tag/become-ohsome/) series
+
+Here you see an example response giving the length of residential roads for a bounding box around the german city Heidelberg.
+
+```json
+{
+    "attribution": {
+        "url": "https://ohsome.org/copyrights",
+        "text": "© OpenStreetMap contributors"
+    },
+    "apiVersion": "1.4.1",
+    "metadata": {
+        "executionTime": 858,
+        "description": "Total length of items in meters.",
+        "requestUrl": "http://localhost:8080/elements/length?bboxes=8.6128,49.3183,8.7294,49.4376&time=2010-01-01/2016-08-01/P2Y2M2D&showMetadata=true&filter=highway=residential%20and%20type:way"
+    },
+    "result": [
+        {
+            "timestamp": "2010-01-01T00:00:00Z",
+            "value": 344220.86
+        },
+        {
+            "timestamp": "2012-03-03T00:00:00Z",
+            "value": 352116.48
+        },
+        {
+            "timestamp": "2014-05-05T00:00:00Z",
+            "value": 351579.81
+        },
+        {
+            "timestamp": "2016-07-07T00:00:00Z",
+            "value": 350577.72
+        }
+    ]
+}
+```
+
+## Services, Clients and Dockerized Version
+
+These services use the ohsome API:
+   
+* [ohsomeHeX](https://ohsome.org/apps/osm-history-explorer/#/amenity_clinic_healthcare_clinic_ptpl/2020-06-01T00:00:00Z/3/0/0)
+* [ohsome dashboard](https://ohsome.org/apps/dashboard/)
+* [ohsome2label](https://github.com/GIScience/ohsome2label)
+* [ohsome quality analyst](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/apps/ohsome-quality-analyst)
+
+This is a list of clients for the ohsome API:
+   
+* [ohsome-py](https://gitlab.gistools.geog.uni-heidelberg.de/giscience/big-data/ohsome/libs/ohsome-py) - Python client
+* [ohsome2x](https://www.npmjs.com/package/@giscience/ohsome2x) - Client that accepts and returns GeoJSON files and PostgreSQL/PostGIS tables.
+* [ohsome-qgis-plugin](https://github.com/MichaelsJP/ohsome-qgis-plugin) - QGIS plugin that supports data extraction
```

Comment:
Shouldn't this tool be in the area of control of HeiGIT first?

Reply:
Well, this is not a list of clients built at HeiGIT per se, so it could stay on the list. IMHO the point is about if it is worth mentioning tools like that. It was updated 6 months ago and we don't know if and how good it works.

Reply:
I agree that we can also include tools/clients/etc. created and maintained by external people (which might not even be the case here :wink: ), perhaps it could make sense to mark them somehow as "external". 

> It was updated 6 months ago and we don't know if and how good it works.

maybe @MichaelsJP can answer that best

Reply:
answer from @MichaelsJP: the QGIS plugin is still work in progress, and currently being worked on. We will add the plugin once it's ready to use.